<launch>
  <arg name="image" default="/hsrb/head_rgbd_sensor/rgb/image_rect_color" />
  <arg name="model" default="$(find object_detector)/models/mask_rcnn_wrs_0223.h5" />
  <arg name="class_labels" default="$(find object_detector)/models/class.txt" />

  <arg name="detection_threshold" default="0.5" />
  <arg name="is_service" default="false" />
  <arg name="debug" default="false" />

  <node pkg="object_detector" type="object_detector_node.py"
	name="object_detector" output="screen" >
    <remap from="image" to="$(arg image)" />
    <param name="model" value="$(arg model)" />
    <param name="class_labels" value="$(arg class_labels)" />

    <param name="detection_threshold" value="$(arg detection_threshold)" />
    <param name="is_service" value="$(arg is_service)" />
    <param name="debug" value="$(arg debug)" />
  </node>
</launch>
